dataikuapi/dss/ml.py

```diff
@@ -1686,15 +1686,17 @@ def redeploy_to_flow(self, model_id, recipe_name=None, saved_model_id=None, acti
             "POST", "/projects/%s/models/lab/%s/%s/models/%s/actions/redeployToFlow" % (self.project_key, self.analysis_id, self.mltask_id, model_id),
             body = obj)
 
-    def guess(self, prediction_type=None):
+    def guess(self, prediction_type=None, redetect=None):
         """
         Guess the feature handling and the algorithms.
         :param string prediction_type: In case of a prediction problem the prediction type can be specify. Valid values are BINARY_CLASSIFICATION, REGRESSION, MULTICLASS.
+        :param bool redetect: if True then the DSS automatically initializes ML task settings, if False, only the target values are remapped
```

Comment:
Should specify that `True` is the default. Can also skip the `True` part, kind of obvious for a boolean.

```suggestion
        :param bool redetect: if False then skip ML task settings automatic detection, only the target values are remapped
```